this is ugly, really these libs should be checked in order

--- grass-6.4.0/configure.in~	2010-08-15 23:34:39.000000000 +0300
+++ grass-6.4.0/configure.in	2010-12-26 22:26:02.982445841 +0200
@@ -514,8 +514,8 @@
 AC_SUBST(CURSES_MAXY)
 AC_MSG_RESULT($CURSES_MAXY)
 
-AC_CHECK_LIB(ncurses, initscr, CURSESLIB=-lncurses, [
-AC_CHECK_LIB(curses, initscr, CURSESLIB=-lcurses, [
+AC_CHECK_LIB(ncursesw, initscr, CURSESLIB="-lncursesw -ltinfow", [
+AC_CHECK_LIB(ncurses, initscr, CURSESLIB="-lncurses -ltinfo", [
     AC_MSG_ERROR([*** Unable to locate curses library.])
     CURSESLIB=
 ])])
